app/views/nsm/steps/work_items/edit.html.erb

```diff
@@ -2,43 +2,82 @@
-          <% work_items.each do |work_item| %>
-            <tr class="govuk-table__row">
-              <td class="govuk-table__cell"><%= check_missing(work_item.work_type) { t("helpers.label.nsm_steps_work_item_form.work_type_options.#{work_item.work_type}") } %></td>
-              <td class="govuk-table__cell"><%= check_missing(work_item.completed_on.present? && format_period(work_item.time_spent)) %></td>
-              <td class="govuk-table__cell"><%= check_missing(work_item.fee_earner) %></td>
-              <td class="govuk-table__cell govuk-table__nowrap">
-                <%= link_to t('.change'), edit_nsm_steps_work_item_path(@form_object.application, work_item_id: work_item.id) %>
-              </td>
-              <td class="govuk-table__cell govuk-table__nowrap">
-                <%= link_to t('.duplicate'), duplicate_nsm_steps_work_item_path(@form_object.application, work_item_id: work_item.id) %>
-              </td>
-              <td class="govuk-table__cell govuk-table__nowrap">
-                <%= link_to t('.delete'), edit_nsm_steps_work_item_delete_path(@form_object.application, work_item_id: work_item.id) %>
-              </td>
-            </tr>
-          <% end %>
-        </tbody>
-      </table>
-    <% end %>
+    <%= govuk_button_to(t('.add_now'), edit_nsm_steps_work_item_path(current_application, work_item_id: Nsm::StartPage::NEW_RECORD), secondary: true) %>
+
+    <%=
+      head = {
+        'line_item' => { html_attributes: { id: 'itemTitle' }, classes: 'govuk-!-width-one-zero' },
+        'item' => {},
+        'date' => {},
+        'fee_earner' => { classes: 'govuk-!-width-one-zero' },
+        'time' => { numeric: true, classes: 'govuk-!-width-one-zero' },
+        **(@form_object.application.allow_uplift? ? { 'uplift' => { numeric: true, classes: 'govuk-!-width-one-zero' } } : {}),
+        'action' => { numeric: true }
+      }
+      header_row = head.map.with_index do |(column_key, options), index|
+        if column_key == 'action'
+          {
+            text: t(column_key, scope: "nsm.steps.view_claim.work_items"),
+            **options
+          }
+        else
+          aria_sort, next_direction = sort_variable(column_key, @sort_by, @sort_direction)
+
+          {
+            text: reorder_form(work_items_nsm_steps_view_claim_path(current_application), column_key, next_direction, "nsm.steps.view_claim.work_items", index),
+            **options.deep_merge(html_attributes: { 'aria-sort': aria_sort})
+          }
+        end
+      end
+
+      rows = @work_items.map do |work_item|
+        item_with_link = link_to(
+                           t("summary.nsm/cost_summary/work_items.#{work_item.work_type.to_s}"),
+                           edit_nsm_steps_work_item_path(@form_object.application, work_item_id: work_item.id),
+                           data: { turbo: 'false' },
+                           'aria-labelledby': "itemTitle item#{work_item.position} workType#{work_item.position}",
```

Comment:
worth stashing the position once here to avoid unneeded calls
```
position = work_item.position
...and use elsewhere in code
```

Reply:
resolved further up thw chain